Solutions/NXLogDnsLogs/Data Connectors/NXLogDnsLogs.json

```diff
@@ -1,8 +1,8 @@
 {
-    "id": "NXLogDnsLogs",

```

Comment:
@v-rbajaj Parser looks good to me, can you please have a look at the other changes once?